source/docs/getting-started/adding-recipe.html.md

```diff
@@ -0,0 +1,136 @@
+
+         1:  #
+         2:  # Cookbook:: git_cookbook
+         3:  # Recipe:: server
+         4:  #
+         5:  # Copyright:: 2017, The Authors, All Rights Reserved.
+         6:  #
+         7:  include_recipe "git_cookbook"
+         8>> include_recipe "runit"
+         9:
+        10:  package "git-daemon-run"
+        11:
+        12:  runit_service "git-daemon" do
+        13:    sv_templates false
+        14:  end
+        15:
+
+       System Info:
+       ------------
+       chef_version=13.0.118
+       platform=ubuntu
+       platform_version=16.04
+       ruby=ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
+       program_name=chef-client worker: ppid=1912;start=15:52:46;
+       executable=/opt/chef/bin/chef-client
+
+
+       Running handlers:
+       [2017-05-08T15:52:47+00:00] ERROR: Running exception handlers
+       [2017-05-08T15:52:47+00:00] ERROR: Running exception handlers
+       Running handlers complete
+       [2017-05-08T15:52:47+00:00] ERROR: Exception handlers complete
+       [2017-05-08T15:52:47+00:00] ERROR: Exception handlers complete
+       Chef Client failed. 0 resources updated in 01 seconds
+       [2017-05-08T15:52:47+00:00] FATAL: Stacktrace dumped to /tmp/kitchen/cache/chef-stacktrace.out
+       [2017-05-08T15:52:47+00:00] FATAL: Stacktrace dumped to /tmp/kitchen/cache/chef-stacktrace.out
+       [2017-05-08T15:52:47+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
+       [2017-05-08T15:52:47+00:00] FATAL: Please provide the contents of the stacktrace.out file if you file a bug report
+       [2017-05-08T15:52:47+00:00] ERROR: Cookbook runit not found. If you're loading runit from another cookbook, make sure you configure the dependency in your metadata
+       [2017-05-08T15:52:47+00:00] ERROR: Cookbook runit not found. If you're loading runit from another cookbook, make sure you configure the dependency in your metadata
+       [2017-05-08T15:52:47+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
+       [2017-05-08T15:52:47+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
+>>>>>> ------Exception-------
+>>>>>> Class: Kitchen::ActionFailed
+>>>>>> Message: 1 actions failed.
+>>>>>>     Converge failed on instance <server-ubuntu-1604>.  Please see .kitchen/logs/server-ubuntu-1604.log for more details
+>>>>>> ----------------------
+>>>>>> Please see .kitchen/logs/kitchen.log for more details
+>>>>>> Also try running `kitchen diagnose --all` for configuration
+~~~
+
+See what the Chef run told you?
+
+>> "Cookbook runit not found. If you're loading runit from another cookbook, make sure you configure the dependency in your metadata"
+
+Hey, that's exactly what we need to do!
+
+<div class="sidebar--footer">
+<a class="button primary-cta" href="/docs/getting-started/adding-dependency">Next - Adding a Dependency</a>
```

Comment:
Counterpoint: we're including inspec and guide has always been pretty Chef specific even if unintended.

I like keeping kitchen neutral but I think including Chef specific information at this point is the happiest path for the most users. However, I do really want to host guides for non-Chef provisioners on the site so that we can do a solid for the entire kitchen community.